frontend/components/search-results.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+import PropTypes from 'prop-types';
+import { BadgeExamples } from './badge-examples';
+import badgeExampleData from '../../badge-examples.json';
+import { prepareExamples, predicateFromQuery } from '../lib/prepare-examples';
+import { baseUri, longCache } from '../constants';
+
+
+export default class SearchResults extends React.Component {
+
+  constructor(props) {
```

Comment:
The `React.Component` constructor sets `props` for you, so you don't need this constructor override.